tools-local/tasks/core-setup.tasks.csproj

```diff
@@ -82,5 +82,7 @@
     <TargetingPackReference Include="Microsoft.Build.Framework" />
     <TargetingPackReference Include="Microsoft.Build.Utilities.v4.0" />
   </ItemGroup>
+
+  <Import Project="$(IntermediateOutputPath)/core-setup.tasks.csproj.nuget.g.targets" Condition="Exists('$(IntermediateOutputPath)/core-setup.tasks.csproj.nuget.g.targets')" />
```

Comment:
?? What's this needed for?

It would be really nice to use an SDK-based project here....

Reply:
This is need to get the netstandard refs. Over lunch I was thinking the same think you mentioned which is to turn this into a sdk project. Let me give that a go.